src/js/compatibility.js

```diff
@@ -72,8 +89,33 @@
                 };
             });
 
+            /*
+                We've removed our browser/OS inspection utils, is* (isChrome, isAndroid, etc.) and have replaced them with
+                an Environment object. This object details the environment in which the player thinks it's in. Refer to our
+                API docs for more information.
+            */
+            var environment = playerInstance.getEnvironment();
+            var utils = playerInstance.utils;
+            var valueFn = function (getter) { return function() { return getter; }; };
+
+            utils.isAndroidNative = valueFn(environment.OS.androidNative);
+            utils.isAndroid = valueFn(environment.OS.android);
+            utils.isChrome = valueFn(environment.Browser.chrome);
+            utils.isEdge = valueFn(environment.Browser.edge);
+            utils.isFF = valueFn(environment.Browser.firefox);
+            utils.isFacebook = valueFn(environment.Browser.facebook);
+            utils.isFlashSupported = valueFn(environment.Features.flash);
+            utils.isIE = valueFn(environment.Browser.ie);
+            utils.isIETrident = function () { return environment.Browser.ie && environment.Browser.version.major >= 11; };
+            utils.isIOS = valueFn(environment.OS.iOS);
+            utils.isIPad = valueFn(environment.OS.iPad);
+            utils.isIPod = valueFn(environment.OS.iPhone);
+            utils.isMSIE = valueFn(environment.Browser.MSIE);
+            utils.isMobile = valueFn(environment.OS.mobile);
+            utils.isOSX = valueFn(environment.OS.OSX);
+            utils.isSafari = valueFn(environment.Browser.safari);
+
```

Comment:
Are you going to put these back in for the time-being to give us more time to update plugins?